lib/google/gax/operation.rb

```diff
@@ -106,11 +106,12 @@ class Operation
       # @param client [Google::Longrunning::OperationsClient]
       #   The client that handles the grpc operations.
       # @param result_type [Class] The class type to be unpacked from the
```

Comment:
Want to change the types list to `[Class, nil]` for these?

Reply:
Updated.